jobs/haproxy/spec

```diff
@@ -27,6 +27,9 @@ properties:
   ha_proxy.internal_only_domains:
     description: "Array of domains for internal-only apps/services (not hostnames for the apps/services)"
     default: []
+  ha_proxy.trusted_domain_cidrs: 
+    description: "Space separated trusted cidr blocks for internal_only_domains"
+    default: 0.0.0.0/0
```

Comment:
I think the default here should be 0.0.0.0/32? As written, it seems like it's defaulting to deny `internal_only_domains` when the source is not coming from a valid IP. Right?